pkg/controller/operconfig/configmap_controller.go

```diff
@@ -0,0 +1,103 @@
+package operconfig
+
+import (
+	"bytes"
+	"context"
+	"log"
+
+	"github.com/openshift/cluster-network-operator/pkg/controller/statusmanager"
+	"github.com/openshift/cluster-network-operator/pkg/names"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+var _ reconcile.Reconciler = &ReconcileConfigMaps{}
+
+// ReconcileConfigMaps watches for updates to specified resources
+type ReconcileConfigMaps struct {
+	client client.Client
+}
+
+// newConfigMapReconciler returns a new reconcile.Reconciler
+func newConfigMapReconciler(client client.Client) reconcile.Reconciler {
+	return &ReconcileConfigMaps{client: client}
+}
+
+// Add adds  when the Manager is Started.
+func AddConfigMapReconciler(mgr manager.Manager, status *statusmanager.StatusManager) error {
+	return addConfigMapReconciler(mgr, newConfigMapReconciler(mgr.GetClient()))
+}
+
+// add adds a new Controller to mgr with r as the reconcile.Reconciler
+func addConfigMapReconciler(mgr manager.Manager, r reconcile.Reconciler) error {
+	// ConfigMap reconciler
+	c, err := controller.New("configmap-controller", mgr, controller.Options{Reconciler: r})
+	if err != nil {
+		return err
+	}
+	err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForObject{})
```

Comment:
Can you configure this to be 
`c.Watch(...), &handler.EnqueueRequestForOwner{OwnerType: &operv1.Network})`

Reply:
Changed.